src/test/java/hudson/plugins/ec2/ConfigurationAsCodeTest.java

```diff
@@ -37,7 +37,7 @@ public class ConfigurationAsCodeTest {
     @Test
     @ConfiguredWithCode("EC2CloudEmpty.yml")
     public void testEmptyConfig() throws Exception {
-        final AmazonEC2Cloud ec2Cloud = (AmazonEC2Cloud) Jenkins.get().getCloud("");
+        final AmazonEC2Cloud ec2Cloud = (AmazonEC2Cloud) Jenkins.get().getCloud("empty");
```

Comment:
This is changing the test intent, is it intentional?

Reply:
Actually we had this discussion previously that was resolved :)  https://github.com/jenkinsci/ec2-plugin/pull/875#discussion_r1277677549

The purpose of the test was to create an ec2Cloud with no templates, so using the name "empty" is still okay. There is a check in core now that prevents the creation of a template with no name.